Source/WebKit/UIProcess/ApplicationStateTracker.mm

```diff
@@ -50,6 +51,28 @@ - (BOOL)_isHostedInAnotherProcess;
 
 namespace WebKit {
 
+static WeakHashSet<ApplicationStateTracker>& allApplicationStateTrackers()
+{
+    static NeverDestroyed<WeakHashSet<ApplicationStateTracker>> trackers;
+    return trackers;
+}
+
+static void updateApplicationBackgroundState()
+{
+    static bool isInBackground = false;
+    auto isAnyStateTrackerInBackground = []() -> bool {
+        return WTF::allOf(allApplicationStateTrackers(), [](auto& tracker) {
```

Comment:
Don't you mean anyOf() here?

Reply:
No, should be changed to areAlllStateTrackersInBackground.